src/main/steps/applicant1/check-your-answers/content.ts

```diff
@@ -57,9 +58,35 @@ const getOtherCourtCasesMoreDetailsContent = () => {
   return moreDetailsComponent({ text, title });
 };
 
+const cannotUploadDocumentList = (
```

Comment:
can we move this method to content.utils

Reply:
its really only used here, do you think its worth moving? i think in the future these functions might just be incorporated with en/cy methods.